data/autoyast_qam/12-common_base_installation.xml.ep

```diff
@@ -245,7 +245,7 @@
     <global>
       <timeout config:type="integer">-1</timeout>
     </global>
-    % if ($check_var->('UEFI', '1') and $check_var->('ARCH', 'x86_64')) {
+    % if ($check_var->('UEFI', '1')) {
```

Comment:
I thought aarch64 had different loader type, but it was probably just different efi package.

Reply:
aarch64 can support `UEFI` only, and it will use the saem loader type as x86_64+UEFI :)